### PR TITLE
Sprint 3 - Planning section

### PR DESCRIPTION
**Pull request by MrHonket**

### CODE REVIEW

File: sections/Chapter 5 Sprint 3/increment1.tex [hunk 1]
```diff
@@ -0,0 +1,128 @@
+\section{Increment 1.}
+This section will cover the progress made in the first increment. 
+
+% --TEORI--
+
+% Docker
+    % hvad kan det
+    % hvorfor bruger vi det
+\subsection{Docker}
+Docker is an open source containerization platform that through the usage of containers allows for a simplification of deliveries in distributed applications.
```
Comment (Pattrigue) [inline sections/Chapter 5 Sprint 3/increment1.tex:10]:
```suggestion
Docker is an open source containerization platform that allows for a simplification of deliveries in distributed applications through the usage of containers. 
```

File: sections/Chapter 5 Sprint 3/increment1.tex [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+% Docker
+    % hvad kan det
+    % hvorfor bruger vi det
+\subsection{Docker}
+Docker is an open source containerization platform that through the usage of containers allows for a simplification of deliveries in distributed applications. 
+Before giving a more detailed description of Docker an overview of what a container is and how it is used is needed. 
+
+
+Containers were originally developed to remove the problem of a piece of system working on one system but having problem when moved to a different one. 
+To tackle this problem software containers were developed.
```
Comment (Pattrigue) [inline sections/Chapter 5 Sprint 3/increment1.tex:15]:
```suggestion
```

File: sections/Chapter 5 Sprint 3/increment1.tex [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+Before giving a more detailed description of Docker an overview of what a container is and how it is used is needed. 
+
+
+Containers were originally developed to remove the problem of a piece of system working on one system but having problem when moved to a different one. 
+To tackle this problem software containers were developed. 
+Using containers allows developers to package their code together with its required dependencies. This package then ensures that the software can be moved reliably between environments with a minimal amount of problems
+
+
+Here a container image is a lightweight version encapsulating everything needed to run that application. These images are then turned into containers at runtime. 
+Mowing around a prepackaged application with all its dependencies insures that the software is going to run the same regardless of the infrastructure in place.
```
Comment (Pattrigue) [inline sections/Chapter 5 Sprint 3/increment1.tex:20]:
```suggestion
```

File: sections/Chapter 5 Sprint 3/increment1.tex [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+
+Containers were originally developed to remove the problem of a piece of system working on one system but having problem when moved to a different one. 
+To tackle this problem software containers were developed. 
+Using containers allows developers to package their code together with its required dependencies. This package then ensures that the software can be moved reliably between environments with a minimal amount of problems
+
+
+Here a container image is a lightweight version encapsulating everything needed to run that application. These images are then turned into containers at runtime. 
+Mowing around a prepackaged application with all its dependencies insures that the software is going to run the same regardless of the infrastructure in place. 
+This is made possibly by the built process isolation and virtualization capabilities in the Linux kernel.
```
Comment (Pattrigue) [inline sections/Chapter 5 Sprint 3/increment1.tex:21]:
```suggestion
This is made possible by the build process isolation and virtualization capabilities in the Linux kernel. 
```

File: sections/Chapter 5 Sprint 3/increment1.tex [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+The result being that containers allows for the same functionality and benefits as VM’s gives us the following advantages:
+
+\begin{itemize}
+    \item Light weight 
+    \item Resource efficiency
+    \item Improved developer productivity
+\end{itemize}
+
+Docker itself it then used to enhance these native linux features allowing us to easier move the containers between environments and automation of container creation.
```
Comment (Pattrigue) [inline sections/Chapter 5 Sprint 3/increment1.tex:35]:
```suggestion
Docker itself is then used to enhance these native Linux features allowing us to automate container creation and easily move the containers between environments.
```

File: sections/Chapter 5 Sprint 3/planning.tex [hunk 6]
```diff
@@ -0,0 +1,48 @@
+\section{Sprint Planning}
+\subsection{Knox Goal}
+The overall goal for the Knox project during this sprint was to get the search engine to work. To accomplish this goal the product owner specified a list of expectation that when completed would fufill a minimal viable product. The specified list is outlined in figure X.
```
Comment (Pattrigue) [inline sections/Chapter 5 Sprint 3/planning.tex:3]:
```suggestion
The overall goal for the Knox project during this sprint was to get the search engine to work. To accomplish this goal, the product owner specified a list of expectations that, when completed, would fufill a minimal viable product. This is captured in figure X.
```
Figure X needs to be replaced with the actual figure.

File: sections/Chapter 5 Sprint 3/planning.tex [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+\begin{itemize}
+	\item It should be possible to make searches in all the available databases.
+	\item It should be possible to make searches using keywords in all the available data.
+	\item It should be possible to make searches in all Nordjyske articles from 2017-2021
+	\item It should be possible to make searches in all the Grundfos manuals, barring those that have shown to be problematic during processing.
+	\item The UI should minimaly be able to display a link to the given article but preferably dispaly the title and text.
+\end{itemize}
+
+With the overall goal for the sprint contained in figure x, a granulated sprint goal could be made, that takes into account the focus of our scrum group.
```
Comment (Pattrigue) [inline sections/Chapter 5 Sprint 3/planning.tex:12]:
```suggestion
With the overall goal for the sprint contained in figure x, a granulated sprint goal could be made that accounts for the focus of our scrum group.
```
Figure x needs to be replaced with an actual figure.

File: sections/Chapter 5 Sprint 3/planning.tex [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+
+\textbf{Sprint goal}
+As noted in the preceding sprint, the code base that was inherited from the previous iteration was deemed problematic and costly to maintain. Therefore our sprint goal for this iteration was based on updating the code base to better comply with the requirements for the Knox goal.
```
Comment (Pattrigue) [inline sections/Chapter 5 Sprint 3/planning.tex:16]:
```suggestion
As noted in the preceding sprint, the codebase that was inherited from the previous iteration was deemed problematic and costly to maintain. Therefore our sprint goal for this iteration was based on updating the codebase to better comply with the requirements for the Knox goal.
```

File: sections/Chapter 5 Sprint 3/planning.tex [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+
+Based on this observation as well as the aforementioned requirements, our sprint goal was as follows:
+
+
+\textit{Make the minimally required amount of API endpoints available, to satisfy the requirements for the search engine, and create the necessary documentation.}
```
Comment (Pattrigue) [inline sections/Chapter 5 Sprint 3/planning.tex:22]:
```suggestion
\textit{Make the minimally required amount of API endpoints available to satisfy the requirements for the search engine, and create the necessary documentation.}
```

File: sections/Chapter 5 Sprint 3/increment1.tex [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+This section will cover the progress made in the first increment. 
+
+% --TEORI--
+
+% Docker
+    % hvad kan det
+    % hvorfor bruger vi det
+\subsection{Docker}
+Docker is an open source containerization platform that allows for a simplification of deliveries in distributed applications through the usage of containers. 
+Before giving a more detailed description of Docker, an overview of what a container is and how it is used is needed.
```
Comment (RasmusHoeyerHansen) [inline sections/Chapter 5 Sprint 3/increment1.tex:11]: "How it is used is needed"

File: sections/Chapter 5 Sprint 3/increment1.tex [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+% Docker
+    % hvad kan det
+    % hvorfor bruger vi det
+\subsection{Docker}
+Docker is an open source containerization platform that allows for a simplification of deliveries in distributed applications through the usage of containers. 
+Before giving a more detailed description of Docker, an overview of what a container is and how it is used is needed. 
+
+
+Containers were originally developed to solve the issue where a program may work on one system but encounter problems when moved to a different one.
```
Comment (RasmusHoeyerHansen) [inline sections/Chapter 5 Sprint 3/increment1.tex:14]: Kilde.

File: sections/Chapter 5 Sprint 3/increment1.tex [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+    % hvad kan det
+    % hvorfor bruger vi det
+\subsection{Docker}
+Docker is an open source containerization platform that allows for a simplification of deliveries in distributed applications through the usage of containers. 
+Before giving a more detailed description of Docker, an overview of what a container is and how it is used is needed. 
+
+
+Containers were originally developed to solve the issue where a program may work on one system but encounter problems when moved to a different one. 
+Using containers allows developers to package their code together with its required dependencies. 
+Moving around a prepackaged application with all its dependencies ensures that the software is going to run the same regardless of the infrastructure in place.
```
Comment (RasmusHoeyerHansen) [inline sections/Chapter 5 Sprint 3/increment1.tex:16]: Kilde

File: sections/Chapter 5 Sprint 3/increment1.tex [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+Docker is an open source containerization platform that allows for a simplification of deliveries in distributed applications through the usage of containers. 
+Before giving a more detailed description of Docker, an overview of what a container is and how it is used is needed. 
+
+
+Containers were originally developed to solve the issue where a program may work on one system but encounter problems when moved to a different one. 
+Using containers allows developers to package their code together with its required dependencies. 
+Moving around a prepackaged application with all its dependencies ensures that the software is going to run the same regardless of the infrastructure in place. 
+
+
+A container image is a lightweight version encapsulating everything needed to run that application. These images are then turned into containers at runtime.
```
Comment (RasmusHoeyerHansen) [inline sections/Chapter 5 Sprint 3/increment1.tex:19]: Kilde

File: sections/Chapter 5 Sprint 3/planning.tex (hunk 8 above)
Comment (RasmusHoeyerHansen) [reply sections/Chapter 5 Sprint 3/planning.tex:16]: Generally, I would like to know how all this ties together with the search engine

**Review by RasmusHoeyerHansen — COMMENTED** [review]
Generally, comma and sources. Also a bit of wonky sentences.

**Pull request merged**